src/app/Library/CrudPanel/Traits/Update.php

```diff
@@ -30,10 +30,14 @@ public function update($id, $data)
 
         // omit the n-n relationships when updating the eloquent item
         $nn_relationships = Arr::pluck($this->getRelationFieldsWithPivot(), 'name');
+        $repeatable_relationships = Arr::pluck($this->getRelationFieldsWithRepeatable(), 'name');
+        $excluded_relationships = array_merge($nn_relationships, $repeatable_relationships);
 
-        $data = Arr::except($data, $nn_relationships);
+        $data = Arr::except($data, $excluded_relationships);
```

Comment:
This could probably be simplified by having a CrudPanel method that gets fields relationships that are saveable. See https://github.com/Laravel-Backpack/CRUD/pull/3200#discussion_r487415554